build2/KoreBuild.ps1

```diff
@@ -0,0 +1,100 @@
+Write-Host -ForegroundColor Green "Starting KoreBuild 2.0 ..."
+
+$RID = "win7-x64";
+if($env:PROCESSOR_ARCHITECTURE -eq "x86") {
+    $RID = "win7-x86";
+}
+
+$ToolsDirectory = $env:KOREBUILD_TOOLS_DIRECTORY
+if(!$ToolsDirectory) {
+    $ToolsDirectory = Join-Path $env:LOCALAPPDATA "Microsoft\AspNetCoreBuildTools\$RID"
```

Comment:
The tools alone end up taking up 50MB in the repo if we do it per-repo... At this point I'm leaning towards saying that's better than having it in a central location, but it is definitely kinda gross. I'll set it back to using a repo-local install for now, but we'll need to look at solutions for shrinking it.

We could consider making MSBuild Core a pre-req and assuming you've got it somewhere (and providing a script to install it if you don't).